.eslintrc.js

```diff
@@ -44,6 +44,8 @@ module.exports = {
     'arrow-body-style': ['warn', 'as-needed'],
     'no-param-reassign': ['error', { props: false }],
     'import/prefer-default-export': 'off',
+    'import/no-extraneous-dependencies': 'off',
```

Comment:
Why we need to add this?

Reply:
If I remember correct, eslint was complaining about `@hospitalrun/components` since it wasn't in the package.json.